api/jobs/params/update-job-parameters.ts

```diff
@@ -0,0 +1,69 @@
+import { BaseParameters } from "../../parameters/index";
+import { SmartlingException } from "../../exception/index";
+
+interface CustomFieldBasicRequest {
+    fieldUid: string,
+    fieldValue: string
+}
+
+export { CustomFieldBasicRequest };
```

Comment:
Looks like this export is not needed. Maybe, the interface could be inlined as well.

Reply:
Removed export.